### DISCUSSION TITLE
Add an indicator for whether or not a crate is available on the playground.

### DISCUSSION THREAD

**Discussion by RpxdYTX** [rust-lang/crates.io#5249] [Feature Requests]
Something like a "Test on Playground" with a link to a sample hello world with the `use` isn't bad. This will make clearer which crates are or aren't usable on the playground, just a little utility tweak.

**Comment by Turbo87** [comment]
I'm not sure I understand what you're suggesting. We already have such a button for the crates that are supported by the playground 🤔

**Reply by RpxdYTX** [reply]
Nvm
